src/components/sign-in.js

```diff
@@ -51,8 +53,10 @@ const SignInForm = () => {
         </div>
         <input type="submit" />
       </form>
+      {user && <Navigate to="/tasks" state={user} replace={true} />}
```

Comment:
Can you wrap user in {}. That way we can put things in state if we need too.

Reply:
And since we're going to be using MUI for the component library, lets use `<Box/>` in place of `<div/>`.